lib/web/databases_test.go

```diff
@@ -432,9 +432,19 @@ func TestHandleSQLServerConfigureScript(t *testing.T) {
 			},
 			assertError: require.Error,
 		},
+		{
```

Comment:
Can you add test cases for other command injection characters:
* `;`
* `"`
* `'`
* `&`
* `|`
* `$`
* `\n`
* `()`
* `

Reply:
@jentfoo I've added URI validation, and we're now escaping the URL when placing it on the PowerShell script.

The use cases you mentioned are split into two tests.